testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/support/support_logging.t.js

```diff
@@ -0,0 +1,122 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+/**
+ * Tests support logging.
+ */
+StartTest(function(t) {
+  var newLoggerName = 'aaa.example',
+      clickAndWaitForStoreToLoad = function(triggerQuery, storeQuery) {
+        return {
+          waitFor: 'StoresToLoad',
+          args: function() { return t.cq1(storeQuery).getStore(); },
+          trigger: { click: triggerQuery },
+          desc: 'Click "' + triggerQuery + '" and wait for "' +
+                storeQuery + '"\'s store to load'
+        };
+      },
+      clickRow = function(query, column, value) {
+        return function(next) {
+          var grid = t.cq1(query),
+              store = grid.getStore(),
+              row = t.getRow(grid, store.find(column, value));
+          t.click(row, next);
+        };
+      },
+      waitForMessageDialogsToClose = {
+        waitFor: 'CQNotFound',
+        args: 'nx-message-notification',
+        desc: 'Wait for message dialogs to close'
+      };
+
+  Ext.merge(t.controller('Message').windowOptions, {
+    slideInDuration: 200,
+    slideBackDuration: 200,
+    autoCloseDelay: 300,
```

Comment:
you probably want to set the durations to -1, delay if you are actually needing to wait to check it (which you probably should) can't be -1, but if you we just ignoring them, then -1 would also be better.